uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -344,5 +345,22 @@ private void handleDoneButton() throws UiObjectNotFoundException {
         }
     }
 
+    @Override
+    public void disableAppThroughSettings(@NonNull final String packageName) {
+        Logger.i(TAG, "Disabling app through settings: " + packageName);
+        launchAppInfoPage(packageName);
+        // This is the id for the disable button
+        handleButtonClick("com.android.settings:id/button2");
+        // Confirm disabling app
+        handleButtonClick("android:id/button1");
```

Comment:
What does button1 do?

Reply:
When you click disable in settings, you get a pop up to confirm. button1 is to confirm the disable.